sdk/storage/storage-blob/src/AppendBlobClient.ts

```diff
@@ -61,8 +98,40 @@ export class AppendBlobClient extends BlobClient {
    *                            pipeline, or provide a customized pipeline.
    * @memberof AppendBlobClient
    */
-  constructor(url: string, pipeline: Pipeline) {
-    super(url, pipeline);
+  constructor(url: string, pipeline: Pipeline);
+  constructor(
+    s: string,
```

Comment:
@jeremymeng 
Why do we have `s` here instead of `connectionString` ?

Reply:
The last one is the constructor implementation signature. `s` here is the first parameter. It could be the first parameter of any of the above three (either `connectionString` or `url`)

Reply:
Oh, thanks. Got it.